### DISCUSSION TITLE
Is there a semantic model for XML comments?

### DISCUSSION THREAD

**Discussion by voroninp** [dotnet/roslyn#66498] [Q&A] [answered]
It seems that xmldoc comments are only represented at Syntax Tree level, but if so, then I have to process `cref` attribute of `see` and `seealso` elements myself. But as VS can do it already, I'd expect there should be some higher level API for this, yet I could not find it.

My task is to generate documentation from metadata captured in attributes, but I'd like to pull xml doc part as well.

**Comment by CyrusNajmabadi** [comment, marked as answer]
Yes. You can use GetSymbolInfo on the nodes inside a cref. That should work already.
